Tests/Admin/AdminHelperTest.php

```diff
@@ -18,6 +18,9 @@
 
 class AdminHelperTest extends \PHPUnit_Framework_TestCase
 {
+    /**
+     * @covers Sonata\AdminBundle\Admin\AdminHelper::getChildFormBuilder
+     */
```

Comment:
> AFAIK, is you write @covers Some::thing and Some::that is called, it would not be covered even if it's tested.

That is exactly the point, calling the constructor for example, does not mean it is tested/covered. If you'd say this is covered you need to explicitly add `@covers \AppBundel\Model\Foo:__construct`

Otherwise we have green lines, but that only means they are executed, not covered